File: 2571-find-the-pivot-integer/find-the-pivot-integer.py
class Solution:
    def pivotInteger(self, n: int) -> int:
        a=[]
        for i in range(1,n+1):
            a.append(i)
        k=-1
        for i in range(n):
            if(sum(a[:i+1])==sum(a[i:])):
                k=i
                break
        if(k!=-1):
            return k+1
        else:
            return k

        

File: 2571-find-the-pivot-integer/find-the-pivot-integer.cpp
class Solution {
public:
    int pivotInteger(int n) {
        vector<int>a;
        for(int i=1;i<=n;i++){
            a.push_back(i);
        }
        for(int i=0;i<a.size();i++){
            int lt=0,rt=0;
            for(int j=0;j<i;j++){
                lt+=a[j];
            }
            for(int j=i+1;j<a.size();j++){
                rt+=a[j];
            }
            if(lt==rt){
                return i+1;
            }
        }
        return -1;
    }
};